README.md

```diff
@@ -347,10 +347,46 @@ az container create -g $He_App_RG --image retaildevcrew/webvalidate:debug -o tsv
 --log-analytics-workspace $(eval $He_LogAnalytics_Id) --log-analytics-workspace-key $(eval $He_LogAnalytics_Key) \
 --command-line "dotnet ../webvalidate.dll --tag southeastasia -l 10000 -s https://${He_Name}.azurewebsites.net -u https://raw.githubusercontent.com/retaildevcrews/${He_Repo}/master/TestFiles/ -f benchmark.json -r --json-log"
 
-# Query logs in Log Analytics (takes several minutes after ACI creation to see logs)
-# TODO: add more query examples?
-az monitor log-analytics query -w $He_LogAnalytics_Id \
---analytics-query "ContainerInstanceLog_CL | sort by TimeGenerated asc "
+```
+
```

Comment:
We should consider simple samples here and a link to detailed samples / field definitions / webv command line. 

I wonder if we can automate saving some sample queries to the LA query in the portal?  That would be a great developer experience. 

Above should be an issue(s) and new PR(s).